### PR TITLE
Prevent remote dependency and trace information telemetry types from being captured

### PR DESCRIPTION
**Pull request by millicentachieng**
Exclude remote dependency and trace information telemetry data from being captured. 

Telemetry types being captured:
- Custom events
- Custom metrics
- Exceptions
- Page views

### CODE REVIEW

**Comment by thewahome** [comment]
@millicentachieng  how do we test this?

**Comment by millicentachieng** [comment]
> @millicentachieng how do we test this?

By logging into the Azure Portal and checking if the excepted telemetry types are being captured.

**Comment by millicentachieng** [comment]
@irvinesunday, @thewahome please review

**Comment by thewahome** [comment]
@millicentachieng what type of data should I look out for? I am not quite sure what to expect

**Comment by millicentachieng** [comment]
> @millicentachieng what type of data should I look out for? I am not quite sure what to expect

@thewahome I've updated the description of the PR to include the list

**Review by thewahome — APPROVED** [review]

File: src/telemetry/telemetry.ts [hunk 1]
```diff
@@ -64,6 +66,15 @@ class Telemetry implements ITelemetry {
     return true;
   }
 
+  private includeSpecifiedTelemetryTypes(envelope: ITelemetryItem) {
+    const baseType = envelope.baseType || '';
+    const typesToInclude = ['EventData', 'MetricData', 'ExceptionData', 'PageviewData'];
+    if (!typesToInclude.includes(baseType)) {
```
Comment (MIchaelMainer) [inline src/telemetry/telemetry.ts:72]: nit: change the condition to make it slightly more readable.

```javascript
if (typesToInclude.includes(baseType)) {
   return true;
}
return false;
```

**Review by MIchaelMainer — APPROVED** [review]

**Pull request merged**